department-of-veterans-affairs/va.gov-cms
# VACMS-3633 Refactor Front end web build

## Description

See #3633. This contains a refactor of how the environment (local/tugboat/brd/devshop) difference are handled.  It also contains calling a composer command for lando and tugboat builds.

## Testing done

* Go to `admin/content/deploy` and see if the messages match the environment.  
* Press the `release content` button.  
  * On lando/tugboat this will take a long time as it builds synchronously.
  * On devshop this will trigger a new devshop tasks.
  * on dev/stage/prod it will trigger a jenkins job:  this can't be tested until merge.
* Go to a node page and make sure the deploy status still work.
* Go to a node page and verify preview link works.  This will work on local also.

![PTSD Treatment | Veterans Affairs 2020-12-09 12-56-26](https://user-images.githubusercontent.com/121603/101668087-1de66380-3a1e-11eb-8b21-99d62adf495b.png)



### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/3677)
